Kosystem/wwwroot/css/paper-nav-fixes.css

```diff
@@ -0,0 +1,65 @@
+﻿
+
+@media only screen and (max-width:768px) {
+    nav.no-collapse {
+        display: flex;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/HelloWorldSweden/Kosystem/pullRequest?prid=5636989)